infrastructure/stacks/application/lambda.tf

```diff
@@ -10,6 +10,7 @@ module "organisations-data-manager-lambda" {
 
   function_name = var.organisations_function_name
   description   = "Microservice for interacting with organisations dynamodb table"
+  # layers        = ["arn:aws:lambda:eu-west-2:017000801446:layer:AWSLambdaPowertoolsPythonV2:46"]
```

Comment:
Does this still need to be here now you've moved it to the module?

Reply:
good spot . No but i think I have written it so you can add specific layers from the stack if needed. I will remove htis. dead code - thanks @tamaragoldschmidt